static/js/home.js

```diff
@@ -185,40 +189,57 @@ function getEvents(){
         'facetObj':JSON.stringify(facetObj)
     };
 
-    if(maptToPlot == "roadsegments") {
-        jQuery.post(
-            "/get_segs",
-            bounddic,
-            getEventCB,
-            'json');
-    }else{
-        jQuery.post(
-            "/get_accidents",
-            bounddic,
-            getEventCB,
-            'json');
-    }
+  if(maptToPlot == "roadsegments") {
+    jQuery.post(
```

Comment:
Given the suggestion below on splitting the callback functions into three functions. Here could call the specific callback function given the mapToPlot. 

Is there a typo here, mapToPlot rather than maptToPlot?